cime_config/tests.py

```diff
@@ -266,6 +266,7 @@
             "SMS_D_Ld1.T62_oQU240wLI.GMPAS-IAF-PISMF.mpaso-impl_top_drag",
             "SMS_D_Ld1.T62_oQU240.GMPAS-IAF.mpaso-harmonic_mean_drag",
             "SMS_D_Ld1.T62_oQU240.GMPAS-IAF.mpaso-upwind_advection",
+            "ERS_D.T62_oQU240.GMPAS-IAF.mpaso-conservation_check",
             )
```

Comment:
could this be ERS_Ld5_D instead? Most of the stealth tests are short because they get run on the cloud

Reply:
Works for me.  @mark-petersen, would you make that change?